pylira/core.py

```diff
@@ -49,7 +49,7 @@ class LIRADeconvolver:
         data = point_source_gauss_psf()
         data["flux_init"] = data["flux"]
         deconvolve = LIRADeconvolver(
-            alpha_init=np.ones(data["psf"].shape[0])
+            alpha_init=np.ones(np.log2(data["counts"].shape[0]).astype(int))
```

Comment:
Indeed my bad...the shape was incorrect here. Maybe adapt the docstring of the `LIRADeconvolver` to describe which shape is expected and add a check to `__init__`? This way we can avoid users pass in the incorrect shape...

Reply:
I added a check in the run function. Can you check it?